formula-requirements.txt

```diff
@@ -1,2 +1,14 @@
-git@github.com:ministryofjustice/repos-formula.git
-git@github.com:ministryofjustice/nginx-formula.git
+git@github.com:ministryofjustice/nginx-formula.git==v3.2.1
+git@github.com:ministryofjustice/supervisor-formula.git==v1.1.0
```

Comment:
Is supervisor-formula still a requirement? I seem to remember munhitsu went and removed many (though maybe not all) of our remaning cases where we pulled it in.